app/views/todos/_form.html.erb

```diff
@@ -1,10 +1,12 @@
-<li id="<%= dom_id(todo) %>" class="todo-list__todo">
-  <%= form_with model: todo, local: false, class: "form" do |form| %>
-    <div class="form__field">
-      <%= form.text_field :name, class: "form__input", autocomplete: :off %>
-    </div>
+<%= turbo_frame_tag dom_id(todo) do %>
```

Comment:
Should work even without `dom_id` according to this. https://github.com/hotwired/turbo-rails/pull/35/files

Reply:
Done in 03b9db28b3ba3bad3958d37f738e4a7da8c25768.